timezones/full/build.gradle.kts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2019-2024 JetBrains s.r.o. and contributors.
+ * Use of this source code is governed by the Apache 2.0 License that can be found in the LICENSE.txt file.
+ */
+
+import com.github.gradle.node.npm.task.NpmTask
+import com.github.gradle.node.npm.task.NpxTask
+import org.jetbrains.kotlin.gradle.targets.js.dsl.ExperimentalWasmDsl
+import org.jetbrains.kotlin.gradle.targets.js.npm.NpmResolverPlugin
+import java.util.*
+
+plugins {
+    kotlin("multiplatform")
+    id("maven-publish")
+    id("com.github.node-gradle.node") version "7.0.2"
+}
+
+val tzdbVersion: String by rootProject.properties
+version = "$tzdbVersion-spi.$version"
+
+val convertedKtFilesDir = File(project.buildDir, "wasmWasi-full/src/internal/tzData")
+val tzdbDirectory = File(project.projectDir, "tzdb")
+
+val timeTzdbInstall by tasks.creating(NpmTask::class) {
+    args.addAll(
+        "install",
+        "@tubular/time-tzdb",
+    )
+}
+
+val tzdbDownloadAndCompile by tasks.creating(NpxTask::class) {
+    tzdbDirectory.mkdirs()
```

Comment:
Please move it to the execution phase, e.g. by wrapping in `doFirst { }`

Reply:
Done